release-notes/major10/10.8.0/valtimo-backend-libraries.md

```diff
@@ -19,9 +19,11 @@ The following features were added:
 
 The following bugs were fixed:
 
-* **Bug1**
+* **Task candidate users**
 
-  Description of what the issue was.
+  The list of assignees for a user task is no longer incomplete. The bug happened when the user task was configured to
+  have more than one role in the candidate groups. The list of assignees would only contain users for the first role in
+  the candidate groups. This has been fixed.
```

Comment:
```suggestion
  the candidate groups.
```

You can remove the last line, it being in this section of the release notes means it's resolved.